### ISSUE TITLE
Firmware update status in DEVICE tab is incorrect due to API change in ISCP protocol

### ISSUE THREAD

**Issue by mkulesh** [mkulesh/onpc#251]
The message UPD in ISCP has changed. 

Previous update status flag was
```
"00" | not exist new firmware
"01" | exist new firmware
"02" | exist new firmware(force)
```

New update status flag is
```
"FF" | not exist new firmware
"00" | exist new firmware
"01" | exist new firmware(Update Notice Normal)
"02" | exist new firmware(Update Notice Force)
```

The message FirmwareUpdateMsg shall be changed accordingly.

**Comment by mkulesh** [comment]
To be merged into free branch

**Comment by mkulesh** [comment]
Released in version 1.26
